docs/14_0_0/components/chart.md

```diff
@@ -0,0 +1,270 @@
+# Charts
+
+Chart.js based components are a modern replacement for the older jQuery-based `p:chart` component. Each chart component has its own model api that defines the data and the options to customize the graph.
+
+[See this widget in the JavaScript API Docs.](../jsdocs/classes/src_PrimeFaces.PrimeFaces.widget.Chart.html)
+
+## Chart
+
+A bar chart provides a way of showing data values represented as vertical bars. It is sometimes used to show trend data, and the comparison of multiple data sets side by side.
+
+### Info
+
+| Name | Value |
+| --- | --- |
+| Tag | barChart
+| Component Class | org.primefaces.component.chart.Chart
+| Component Type | org.primefaces.component.Chart
+| Component Family | org.primefaces.component |
+| Renderer Type | org.primefaces.component.ChartRenderer
+| Renderer Class | org.primefaces.component.barchart.ChartRenderer
+
+### Attributes
+
+| Name | Default | Type | Description | 
+| --- | --- | --- | --- |
+| id | null | String | Unique identifier of the component
+| rendered | true | Boolean | Boolean value to specify the rendering of the component, when set to false component will not be rendered.
+| ariaLabel | title | String | The aria-label attribute is used to define a string that labels the current element for accessibility. (default to chart title).
+| binding | null | Object | An el expression that maps to a server side UIComponent instance in a backing bean
+| extender | null | String | Name of JavaScript function to extend the options of the underlying Chart.js plugin.
+| style | null | String | Inline style of the component.
+| styleClass | null | String | Style class of the component.
+| value | null | String | XDEV Model object of data and settings or raw JSON string.
+| widgetVar | null | String | Name of the client side widget.
+
+### Getting Started with Raw JSON
+
+You can pass any full Chart.js configuration using a raw JSON string of data and configuration. This allows you complete and total
+control over the chart and how it is rendered as well as any new Chart.js features as they come along.
+
+```xhtml
+<p:chart value="#{chartView.json}">
+```
+
+```java
+public class Bean {
+    
+    private String json;
+
+    @PostConstruct
+    public void init() {
+        json = "{\r\n"
```

Comment:
can't we use a text block for this (e.g """) ?

Reply:
I would love to but I think that is Java 17.

Reply:
ah my bad, I had in mind it was java 11...

Reply:
For the sake of readibility, and as it is only documentation, here using text blocks would be better

Reply:
Good idea